opentelemetry-sdk/Cargo.toml

```
@@ -53,6 +53,7 @@ rt-tokio-current-thread = ["tokio", "tokio-stream", "experimental_async_runtime"
 internal-logs = ["opentelemetry/internal-logs"]
 experimental_metrics_periodicreader_with_async_runtime = ["metrics", "experimental_async_runtime"]
 spec_unstable_metrics_views = ["metrics"]
+experimental_custom_metric_reader = ["metrics"]
```

We have been following this naming convention: `experimental-{signal}-{feature}`

```suggestion
experimental_metrics_custom_reader = ["metrics"]
```